.github/workflows/npm_publish.yml

```diff
@@ -0,0 +1,29 @@
+name: NPM publish on new release
+
+on: 
+  release:
+    types: [created]
+    branches: 
+      - master
+
+jobs:
+  npm-publish:
+    
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        registry-url: https://registry.npmjs.org/
+    - run: npm ci
+    - run: npm run build
```

Comment:
Just for my edification, we're not including the contents of `src` or `test` in what we publish to NPM, right? We're essentially just publishing the `dist` directory and some manifest files (`package.json`, etc.)

Reply:
Currently we include everything except the things specified in .npmignore. In answers-core, we instead use the 'files' param in package.json to specify which files and folders to include. I think we should switch answers-search-ui to do that as well. I think the `dist` folder is the only one we need

Reply:
I made the change in package.json to include dist/ folder only (npm will also automatically include package.json, license, and readme file when publish)

Reply:
With the `files` addition to the `package.json` do we still need `.npmignore`?

Reply:
Good point. Would need it if we want to exclude something specific from `files`, but since node_modules is automatically ignored and DS_Store is never going to be push to repo, I will remove `.npmignore`